### ISSUE TITLE
Plotting training vs validation loss curve

### ISSUE THREAD

**Issue by cowwoc** [jdb78/pytorch-forecasting#614]
I am trying to diagnose why TFT is outputting a straight line as others have reported (e.g. https://github.com/jdb78/pytorch-forecasting/issues/454). I think plotting the training vs validation loss curves on the same graph would go a long way towards diagnosing these problems but I'm new to pytorch-lighting so I'm not sure how to get this done.

Please take a look at https://stackoverflow.com/questions/66290662/how-to-get-step-wise-validation-loss-curve-over-all-epochs-in-pytorch-lightning. Can you please provide a code snippet (and library support) for plotting this curve across epoch?
